.environment-example

```diff
@@ -26,18 +26,9 @@ export TRELLO_API_KEY=""
 
 export SENTRY_DSN=""
 
-export JOBS_EMAIL_USER=""
-export JOBS_EMAIL_PASSWORD=""
-
-export MEETUPS_EMAIL_USER=""
-export MEETUPS_EMAIL_PASSWORD=""
-
 export GAPPS_PRIVATE_KEY_ID=""
 export GAPPS_PRIVATE_KEY=""
 
-export STRIPE_PUBLIC_KEY=""
```

Comment:
STRIPE keys are still needed  for our donations form so you need to put them back.